services/distribution/src/test/java/app/coronawarn/server/services/distribution/assembly/appconfig/validation/ExposureConfigurationValidatorTest.java

```diff
@@ -72,7 +72,9 @@ private ValidationResult getResultForTest(TestWithExpectedResult test)
   private static Stream<Arguments> createFailedTests() {
     return Stream.of(
         ScoreTooHigh(),
-        ScoreNegative(),
+        // TODO adjust test when new validation for yaml-Files is implemented (check that no attributes are missing
```

Comment:
I've created the ticket now so please update to:
// TODO cwa-server/#320 Validate that no attributes are missing in .yaml

Reply:
Updated